app/src/components/common/SessionFrame.vue

```diff
@@ -65,15 +65,27 @@ export default {
   },
   methods: {
     goBack() {
-      this.$router.go(`-1`)
+      if (location.href.includes(`portfolio`)) {
+        this.$router.push({
+          name: "networks"
```

Comment:
why not always `this.$router.go(`-1`)`?

Reply:
because then you get a double navigation error when you are already on portfolio (CardSignInrequired) and try to navigate to portfolio again

Reply:
ah yeah, but why this happens :thinking: I will recheck